src/app/send/access.component.html

```diff
@@ -58,8 +63,15 @@
                             <i class="fa fa-download" aria-hidden="true"></i>
                             {{'downloadFile' | i18n}} ({{send.file.sizeName}})</button>
                     </ng-container>
+                    <p *ngIf="expirationDate" class="text-center text-muted">Expires: {{expirationDate | date: 'medium'}}</p>
                 </div>
             </div>
         </div>
+        <div class="col-12 text-center mt-5 text-muted">
+            <p class="mb-0">{{'sendAccessTaglineProductDesc' | i18n}}<br>
+            {{'sendAccessTaglineLearnMore' | i18n}} <a href="https://www.bitwarden.com/products/send">Bitwarden Send</a> {{'sendAccessTaglineOr' | i18n}} <a href="https://vault.bitwarden.com/#/register">{{'sendAccessTaglineSignUp' | i18n}}</a> {{'sendAccessTaglineTryToday' | i18n}}
```

Comment:
Change URL for learn more to: `https://www.bitwarden.com/products/send?source=web-vault`